kubernetes/zenko/charts/cloudserver/values.yaml

```diff
@@ -57,12 +57,12 @@ externalBackends:
     keepAlive: false
     keepAliveMsecs: 1000
     maxFreeSockets: 256
-    maxSockets: null
+    maxSockets:
```

Comment:
Double checked to validate this still gets templated as `null` by the entry point into the `config.json`